### DISCUSSION TITLE
🤔 [QUESTION] 仪表盘如何显示指定的刻度 ？

### DISCUSSION THREAD

**Discussion by xiecg** [antvis/G2Plot#3580] [Q&A] [answered]
### 🏞 实现下面图片的效果，文档没找到相关的配置

<img width="572" alt="image" src="https://user-images.githubusercontent.com/10857932/217412222-532e6761-0723-46f6-8f3b-ef5598ca5c9c.png">

**Comment by zhangzhonghe** [comment]
@xiecg 试一下下面这个配置项。

![image](https://user-images.githubusercontent.com/38434641/217420522-f3e80166-09b6-467c-92dc-bc693be413b8.png)

**Comment by xiecg** [comment]
> @xiecg 试一下下面这个配置项。
> 
> ![image](https://user-images.githubusercontent.com/38434641/217420522-f3e80166-09b6-467c-92dc-bc693be413b8.png)

不行，tickLine 出现的位置被固定了(0, 20, 40, 60, 80, 100)，我希望有配置可以指定出现的位置。

**Comment by hustcc** [comment, marked as answer]
自定义 tickMethod 即可，返回你想显示的值。

```ts
axis: {
  label: {
    formatter(v) {
      return Number(v) * 100;
    },
  },
  tickMethod: () => [0.5, 0.7], // 👈🏻
},
```

![image](https://github.com/antvis/G2Plot/assets/7856674/d92b4373-7e56-4cfa-9828-0d731c0119f7)
